cellxgene_schema_cli/cellxgene_schema/ontology.py

```diff
@@ -105,7 +106,22 @@ def get_symbol(self, gene_id) -> str:
         if not self.is_valid_id(gene_id):
             raise ValueError(f"The id '{gene_id}' is not a valid ENSEMBL id for '{self.species}'")
 
-        return self.gene_dict[gene_id]
+        return self.gene_dict[gene_id][0]
```

Comment:
can you elaborate on this change? I'm not seeing compensatory changes where it is called, so I'm assuming this is an unrelated bug fix? Does it need a test?

Reply:
Its related, and tested in `test_ontology` as well as `test_validate`! We previously only mapped gene_id to a single value in the gene_dict (gene_label). Now that we also want to be able to fetch the gene length by ID, I updated the gene_dict to map to a tuple of (gene_label, gene_length), which we derive from the csv fixtures.

accompanying change to populating the gene_dict is here: https://github.com/chanzuckerberg/single-cell-curation/pull/624/files/fb60f0baba71b9d22da47dd6d8708ba16d75322f#diff-e03634d528c86621165785d45334e8b7255794f658cb01e118edd1f6ce817df9R71

Reply:
ah makes sense. Thanks